### PR TITLE
Added user guide for salesforce integration

### PR DESCRIPTION
**Pull request by KSCooke**
Added a Salesforce Integration guide. The guide is a simple example of how to integrate Salesforce into Workforce.

### CODE REVIEW

File: user_guides/salesforce_integration/README.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
+# Salesforce to Workforce Integration Using Oauth 2.0
+
+---
+
+## Creating the connection
+
+This section will help you create an Oauth connection between Salesforce and Workforce, allowing you to send and receive callbacks from [Tanda API](https://my.tanda.co/api/v2/documentation)
+
+### 1. Setup the Auth Provider in Salesforce (Setup > Identity > Auth Provider)**
```
Comment (ghiculescu) [inline user_guides/salesforce_integration/README.md]:
```suggestion
### 1. Setup the Auth Provider in Salesforce (Setup > Identity > Auth Provider)
```

File: user_guides/salesforce_integration/README.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+Enter in dummy data in the key and secret fields so you can create the provider (you will get these after you setup the Workforce Oauth application)
+    ![Example of SalesforceAuthProvider](assets/SalesforceAuthProviderExample.png)
+    ![Example of CallbackUrl](assets/SalesforceCallbackExample.png)
+
+### 2. Setup the Oauth Application in Workforce at the [My Workforce Applications](https://my.workforce.com/api/oauth/applications) page**
```
Comment (ghiculescu) [inline user_guides/salesforce_integration/README.md]:
```suggestion
### 2. Setup the Oauth Application in Workforce at the [My Workforce Applications](https://my.workforce.com/api/oauth/applications) page
```

File: user_guides/salesforce_integration/README.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+Use the Callback URL from above to create your application
+Get the Application ID and Application Secret, and add them to the Salesforce Auth Provider you created
+    ![Tanda Oauth Application](assets/TandaOauthApplicationExample.png)
+
+### 3. Setup Named Credentials using your new Auth Provider.(Setup > Security > Named Credentials)**
```
Comment (ghiculescu) [inline user_guides/salesforce_integration/README.md]:
```suggestion
### 3. Setup Named Credentials using your new Auth Provider.(Setup > Security > Named Credentials)
```

**Review by ghiculescu — COMMENTED** [review]
Thanks!

**Comment by KSCooke** [comment]
Missed removing those bold tags thanks for catching that.

**Review by ghiculescu — APPROVED** [review]

**Pull request merged**